templates/zerver/help/restrict-user-email-addresses-to-certain-domains.md

```diff
@@ -0,0 +1,32 @@
+# Restrict user email addresses to certain domains
+
+Only administrators can restrict user email addresses to certain domains.
+ This can be done through the administration page.
+
+{!go-to-the.md!} [Organization settings](/#administration/)
+{!admin.md!}
+
+2. Restricting user email addresses to certain domains can be enabled or disabled
+ with the **Users restricted to (allowed domains)** checkbox shown below:
+
+    ![Restrict domains checkbox](/static/images/help/restrict-domains-checkbox.png)
+
+    !!! tip ""
+        The allowed domains will be listed after the **Users restricted to**
+         next to the checkbox.
+
+3. Domains can be added to the allowed domains list by clicking on the
+ **Change domain** link next to the checkbox. The **Allowed Domains**
+ list will now appear. You can then add a domain to the list by typing the
+ domain into the box underneath the list and clicking the **Add** button.
+
+4. Domains can be deleted from the **Allowed Domains** list by clicking
+ the **Delete** button (shown below) next to a on the list.
+
+    ![Allowed domain delete button](/static/images/help/allowed-domain-delete.png)
+
+    !!! warn ""
+        Note: You must have at least one allowed domain if
+         **Users restricted to** is enabled
```

Comment:
You don't, you can remove all domains off the list and leave the checkbox activated.